Character.cpp

```diff
@@ -86,32 +88,45 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
-
-#include <assert>
 void Character::attackInternal(Character& other)
 {
-    if( other.hitPoints <= 0 )
+    if(other.hitPoints <= 0)
     {
         /*
         When you defeat another Character: 
             a) your stats are restored to their initial value
             b) your stats are boosted 10%
             c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
       */
-        assert(false);
+        //assert(false);
+        // a)
+        hitPoints = *initialHitPoints;
+        armor = *initialArmorLevel;
+        attackDamage = *initialAttackDamage;
+        // b)
+        hitPoints *= 0.1;
+        armor *= 0.1;
+        attackDamage *= 0.1; 
```

Comment:
this translates to:
```
hitPoints = hitPoints * 0.1;
```
this is not boosting the stat by 10%.  this is **SETTING** the stat to 10% of its value.
- [ ] fix according to the spec.